molecule/default/INSTALL.rst

```diff
@@ -1,16 +0,0 @@
-*******
-Install
```

Comment:
@planet-winter  why is this file deleted?

Reply:
I have no clue... ;-)